### PR TITLE
Predicate Clauses amendment (multiple subjects due to copular clauses)

### PR DESCRIPTION
**Pull request by nschneid**

### CODE REVIEW

**Comment by nschneid** [comment]
Still have to work on the pages for individual relations but welcoming comments on what's written so far.

**Comment by nschneid** [comment]
Do we want to list `nsubj:outer` and `csubj:outer` as "semi-mandatory" on https://universaldependencies.org/u/dep/ (https://github.com/UniversalDependencies/docs/blob/multiple-subjects/_u-overview/dep-index.md)?

**Comment by nschneid** [comment]
@amir-zeldes @dan-zeman if no objections I'll merge this tonight

File: _en/dep/cop.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ nsubj(yesterday, It)
 cop(yesterday, was)
 ~~~
 
-Prepositions may also project a cop dependent.
+Prepositions may also project a `cop` dependent.
```
Comment (amir-zeldes) [inline _en/dep/cop.md:45]: I don't think this is a preposition, more likely an adverb or a predicative adjective (paradigmatic with "the light is bright", not "* the light is for")

Comment (nschneid) [reply _en/dep/cop.md:45]: Would a better example be "the city I'm in"? (cop dependent in relative clause)

Comment (amir-zeldes) [reply _en/dep/cop.md:45]: Sure, that truly is a preposition, though in this case the reason it governs a cop is promotion. I can't think of a legitimate case where a true preposition governs a cop, not due to promotion, and not due to metalinguistic usage. If it has an object (in constituent/SUD terms), then that object would govern the cop in UD, and if it doesn't have an object, then it's not really a preposition, just maybe a homophone of one.

Comment (nschneid) [reply _en/dep/cop.md:45]: You can go ahead and edit the page, either change it to the relative clause or remove it entirely

**Pull request merged**